### DISCUSSION TITLE
looking for a coding mentor

### DISCUSSION THREAD

**Discussion by SeanJ-Source** [community/community#131620] [Programming Help]
### Body

Hello, Im trying to change career paths and go into the tech industry, I get the gist of HTML and CSS for the most part and know a little bit of javascript, but I can and need to get better if I am to make this career change, I am looking for a mentor who can help me figure it out better, Im a fast learner, Ive just been teaching myself so I dont really know where to go from here, hope someone can help me, if not, Ill just keep on keeping on

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Bhargav-92** [comment]
https://www.w3schools.com/

**Comment by Chamal1120** [comment]
Follow this 100% free of cost online path

1. Learn a little bit of math before you begin. I recommend Algebra I and 2 playlist by Professor Dave in YouTube.
2. Then move to cs50x by harvard University. Search on YouTube how to take harvard cs50 for free.
3. Then take the cs50 web extension course.
4. After that if you want you can do other cs50 extension courses also like cs50 game, AI and etc.
5. Then you will have enough knowledge to choose your path and advance on it yourself.
6. Plus point - Try to contribute to open source project if you can along the way.
